src/org/thoughtcrime/securesms/qr/RegistrationQrActivity.java

```diff
@@ -77,6 +79,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case android.R.id.home:
                 finish();
                 return true;
+            case R.id.troubleshooting:
+                // once help is evolved, it may be a more visible button, may be localized, may be offline
+                WebViewActivity.openUrlInBrowser(this, "https://delta.chat/en/help#multiclient");
```

Comment:
these kind of comments tend to get stale :)

also, i am not sure if a more visible button is helpful, as it usually should just work - and if not, the kebab menu is mostly only other option user can tap, so they will find it. more probable is that we add "Paste" to the menu.

i agree, with the offline help, but we have that in mind anyways and won't be reminded by a commented somewhere in the code that is read randomly.


```suggestion
                WebViewActivity.openUrlInBrowser(this, "https://delta.chat/en/help#multiclient");
```

Reply:
I just copy pasted the same comment from the other place where this is called like this, so the comment doesn't get even more stale :D

Reply:
:D